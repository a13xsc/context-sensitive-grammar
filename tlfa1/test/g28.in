 (             {            X ,   O   ,  G          ,       V         ,    ! ,    o ,J, F        , P   ,   A          ,  Z   ,      8 ,         K   ,  i       ,        S     ,          I,       T       ,R       ,      Q       ,          E                  }                 ,               {             ! ,         i         ,   o      ,         8      }         ,              {                  (          G     , VPGREOQR8Z     ) , (     I, SF!SoVOS8  )    ,       (   J         ,      TEoiVPZJTIQ          )   ,(    J       ,          TSJJJZ )    ,       (          J         ,     e  )              }          ,        J          )     
