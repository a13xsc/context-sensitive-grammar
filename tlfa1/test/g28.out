I
A
G
R
E
S
V
T
F
X
O
Q
Z
K
P
